private/bufpkg/bufmodule/bufmoduleref/validate.go

```diff
@@ -167,7 +167,7 @@ func ValidateRemoteNotEmpty(remote string) error {
 // we do not think will change in the future.
 func ValidateRemoteHasNoPaths(remote string) error {
 	_, path, ok := strings.Cut(remote, "/")
-	if ok && path != "" {
```

Comment:
What changed here?

Reply:
I noticed during testing that if you allow the user to have a trailing slash, then when printing out results (such as repositories), they display with an extra slash:

```
± % go run ./cmd/buf beta registry repository list buf.build/                                                                                                                                                                                                                                                                                                                     
Full name                              Created
buf.build//beta/googleapis              2021-02-09T19:26:43Z
buf.build//beta/protoc-gen-validate     2021-02-09T19:26:50Z
```

We could allow the slash and then modifier the printer to remove any trailing slashes, but it seemed less invasive to simply prohibit the trailing slash.  Although if you think that could be considered a breaking change, I can put it back in and fix the printer.

Reply:
I'm not sure how many people this would affect, but it could be breaking for _some._ It's probably simpler to leave this part of the change out and isolate it to its own PR (whether we do a printer change, or the change you've proposed here). That way it'll also be easier for us to have an isolated review there and rollback a single change if we ever need to.

Plus, this seems kinda orthogonal to the primary goal of this PR, so it's probably best to just revert it to what we had before.